tembo-operator/src/app_service/types.rs

```diff
@@ -76,6 +76,28 @@ pub struct Routing {
     pub ingress_path: Option<String>,
     // provide name of the middleware resources to apply to this route
     pub middlewares: Option<Vec<String>>,
+    #[serde(rename = "entryPoints")]
+    #[serde(default = "default_entry_points")]
+    pub entry_points: Option<Vec<String>>,
+    #[serde(rename = "ingressType")]
+    #[serde(default = "default_ingress_type")]
+    pub ingress_type: Option<IngressType>,
+}
+
+#[allow(non_camel_case_types)]
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize, ToSchema, JsonSchema)]
+pub enum IngressType {
+    http,
+    tcp,
+    udp,
```

Comment:
`udp` doesn't do anything for now. We could drop this and add in the future if we like.

Reply:
lets drop it if it doesnt do anything